crates/web-sys/src/features/gen_ClipboardItem.rs

```diff
@@ -34,4 +34,11 @@ extern "C" {
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `ClipboardItem`*"]
     pub fn get_type(this: &ClipboardItem, type_: &str) -> ::js_sys::Promise;
+    # [wasm_bindgen (static_method_of = ClipboardItem , js_class = "ClipboardItem" , js_name = supports)]
+    #[doc = "The `supports()` method."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/supports)"]
```

Comment:
That link seems to be broken, shouldn't it be https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/supports_static ?

Reply:
The link is generated, so that would have to be fixed in the generator.
It would be interesting to see if that is an exception for this specific static method or if this actually fixes a bug in the generator.

Reply:
Addressed in #4010.